BookPlayer/Library/ItemListViewController.swift

```diff
@@ -380,6 +412,27 @@ extension ItemListViewController {
             self.tableView.contentInset.bottom = rootViewController.miniPlayerIsHidden ? 0.0 : 88.0
         }
     }
+
+    @objc func selectButtonPressed(_ sender: Any) {
+        if self.tableView.numberOfRows(inSection: Section.data.rawValue) == (self.tableView.indexPathsForSelectedRows?.count ?? 0) {
+            for row in 0..<self.tableView.numberOfRows(inSection: Section.data.rawValue) {
+                self.tableView.deselectRow(at: IndexPath(row: row, section: .data), animated: true)
+            }
+
+            self.bulkControls.moveButton.isEnabled = false
```

Comment:
these two lines aren't needed since it's already set in `self.updateSelectionStatus()`